doc/logreg.txt

```diff
@@ -264,6 +264,20 @@ approximately 1.936 epochs/sec and it took 75 epochs to reach a test
 error of 7.489%. On the GPU the code does almost 10.0 epochs/sec. For this
 instance we used a batch size of 600.
 
+
+Prediction Using a Trained Model
++++++++++++++++++++++++++++++++
+
+``sgd_optimization_mnist`` serialize and pickle the model each time new
+lowest validation error is reached. We can reload this model and predict
+labels of new data. ``predict`` function shows an example of how
+this could be done.
+
+.. literalinclude:: ../code/logistic_sgd.py
+  :start-after:  ' ran for %.1fs' % ((end_time - start_time)))
```

Comment:
There is a way to specify a function I think. This is more robust. Can you check that?